.code-samples.meilisearch.yaml

```
@@ -18,7 +18,7 @@ delete_an_index_1: |-
   // OR
   $client->index('movies')->delete();
 get_one_document_1: |-
-  $client->index('movies')->getDocument(25684);
+  $client->index('movies')->getDocument(25684, (new DocumentsQuery())->setFields(['id', 'title', 'poster', 'release_date']));
```

I think I was not so clear in my last message, sorry! :| 

This function `getDocument` accepts a plain array in the fields :)
```suggestion
  $client->index('movies')->getDocument(25684, ['id', 'title', 'poster', 'release_date']);
```